src/lib/draw-and-pick.js

```diff
@@ -71,8 +71,9 @@ export function pickLayers(gl, {
   // And compensate for pixelRatio
   const pixelRatio = typeof window !== 'undefined' ?
     window.devicePixelRatio : 1;
-  const deviceX = x * pixelRatio;
-  const deviceY = gl.canvas.height - y * pixelRatio;
+  const deviceX = Math.round(x * pixelRatio);
+  const deviceY = Math.round(gl.canvas.height - y * pixelRatio);
+  const deviceRadius = Math.round(radius * pixelRatio);
```

Comment:
Ok, reading further I see you're using a box despite the name `radius` here. Maybe note here that `radius` really means "half-size"? Or maybe it's fine.